lib/view/ui/userdatawizard/personalinfo_page.dart

```diff
@@ -16,14 +20,26 @@ class PersonalInfoPage extends StatelessWidget {
   PersonalInfoPage({super.key});
 
   final formKey = GlobalKey<FormState>();
+  final FirebaseService _firebaseService = FirebaseService();
+  final FirebaseStorageService storageService = FirebaseStorageService();
+
+  Map<String, dynamic> collectUserData() {
+    final firstName = controller.firstNameController.text;
+    final lastName = controller.lastNameController.text;
+    final email = controller.emailController.text;
+    final displayName = '$firstName $lastName'; // Combine first and last name
+
+
```

Comment:
it should retrun data Model class like 

class USerData{
String  fname, String lastname
}

Reply:
yes the model class final userData = collectUserData(); is there in the onpressed callback function and the function collectUserData returns as 
 return {
      'displayName': displayName,
      'email': email,
    };